src/pages/v2/index.md

```diff
@@ -0,0 +1,76 @@
+---
+keywords:
+  - Express Embed SDK
+  - Full editor
+  - Editor
+  - Embed SDK
+  - Quick actions
+  - SDK
+  - JavaScript
+  - Embed
+title: Adobe Express Embed SDK
+description: This is the overview page of the Adobe Express Embed SDK
+contributors:
+  - https://github.com/amandahuarng
+  - https://github.com/pklaschka
+---
+
+<Hero slots="heading, text" background="rgb(64, 34, 138)" />
+
+# Adobe Express Embed SDK
+
+Adobe Express Embed SDK is an easy-to-integrate, customizable, all-in-one JavaScript library that brings Adobe Express capabilities to all applications. This SDK is currently in open beta and is subject to changes until GA availability.
```

Comment:
Are we still in open beta , just curious?

Reply:
technically public since we have  7 partners live but we're keeping the beta label before X migration